### PR TITLE
JBOwnable

### PR DESCRIPTION
**Pull request by xBA5ED**

### CODE REVIEW

File: contracts/JBTiered721Delegate.sol [hunk 1]
```diff
@@ -28,7 +28,7 @@ import './structs/JBTiered721Flags.sol';
   Votes: A helper for voting balance snapshots.
   Ownable: Includes convenience functionality for checking a message sender's permissions before executing certain transactions.
 */
-contract JBTiered721Delegate is JB721Delegate, Ownable, IJBTiered721Delegate, IERC2981 {
```
Comment (mejango) [inline contracts/JBTiered721Delegate.sol:31]: why move Ownable to the superclass? Can we keep this as JBOwnable here @xBA5ED ?

**Review by mejango — CHANGES_REQUESTED** [review]

File: contracts/JBTiered721Delegate.sol (hunk 1 above)
Comment (mejango) [reply contracts/JBTiered721Delegate.sol:31]: @xBA5ED

**Comment by mejango** [comment]
lets undraft this PR!

**Comment by mejango** [comment]
@xBA5ED can we also add a `requiresPermission` modifier to `adjustTiers` so that a project owner can give an Operator permission to do so on their behalf?

**Review by mejango — APPROVED** [review]

**Review by viraj124 — APPROVED** [review]

**Pull request merged**